processor/deltatocumulativeprocessor/internal/maybe/option.go

```diff
@@ -0,0 +1,25 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package maybe // import "github.com/open-telemetry/opentelemetry-collector-contrib/processor/deltatocumulativeprocessor/internal/maybe"
```

Comment:
This is a new package with utilities for this module. While the code is straightforward, can you please add tests and perhaps a few examples as well? A doc.go would also be helpful. Perhaps this can then, in the future, be moved to /pkg?